zookeeper-server/src/main/java/org/apache/zookeeper/server/persistence/TxnLogToolkit.java

```diff
@@ -361,6 +371,40 @@ static String getFormattedTxnStr(Record txn) {
         return txnData.toString();
     }
 
+    private static Record deserializeTxn(Txn txn) throws IOException {
```

Comment:
Rename to `deserializeSubTxn` ? I was confused until comparing it with `SerializeUtils.deserializeTxn`.